fe/fe-core/src/main/java/org/apache/doris/nereids/stats/FilterEstimation.java

```diff
@@ -117,13 +117,14 @@ public Statistics visitCompoundPredicate(CompoundPredicate predicate, Estimation
         Expression leftExpr = predicate.child(0);
         Expression rightExpr = predicate.child(1);
         Statistics leftStats = leftExpr.accept(this, context);
-        leftStats = leftStats.normalizeByRatio(context.statistics.getRowCount());
-        Statistics andStats = rightExpr.accept(this,
-                new EstimationContext(leftStats));
+        leftStats.normalizeColumnStatistics(context.statistics.getRowCount());
```

Comment:
normalizeByRatio and enforceValid are different.

Reply:
the normalizeByRatio policy can be discussed later...